style.css

```diff
@@ -0,0 +1,49 @@
+
+body {
+  font-family: 'Poppins', sans-serif;
+  margin-left: 5%;
+  margin-right: 5%;
+  cursor: pointer;
+}
+
+@media (min-width: 375px) {
+  .nav-bar {
+    display: flex;
+    justify-content: space-between;
+    font-style: normal;
+    font-size: 112.5%;
+    line-height: 125%;
+    text-align: center;
+    background-color: white;
+    color: #6070ff;
+  }
+
+  .intro {
+    padding-top: 10%;
+    background-image: url(shapes.svg);
```

Comment:
- [ ] The last thing I'll suggest is that currently, your background image looks like it has been cut short from the ones at the edge of the design. But from the given design you can see that we can view the complete background design so kindly ensure that this is the same for your design as well :wink: 

| Your Design | The Figma Design |
|--------|--------|
| ![image](https://github.com/CrackedYoda/MyPortfolio/assets/102699958/08801078-f275-4a49-82fa-a963e86fbd5f) | ![image](https://github.com/CrackedYoda/MyPortfolio/assets/102699958/a5911d7c-01ca-4a5a-8ea0-97c300f3b933) |